Source/OxyPlot/Series/AreaSeries.cs

```diff
@@ -33,14 +33,25 @@ public class AreaSeries : LineSeries
         /// </summary>
         private List<DataPoint> actualPoints2;
 
-        /// <summary>
-        /// Initializes a new instance of the <see cref = "AreaSeries" /> class.
-        /// </summary>
-        public AreaSeries()
+		/// <summary>
+		/// The index of the data item at the start of visible window
+		/// </summary>
+		private int winIndex;
```

Comment:
how much performance do you gain by persisting this? I am concerned about premature optimization...